index.html

```diff
@@ -64,5 +64,196 @@ <h2 class="intro connect">LET'S CONNECT</h2>
         </li>
       </ul>
     </section>
+    <!-- Work Section -->
+    <section class="wrapper">
+      <div class="shot">
+        <div class="snapshot"><img src="images/Shot1.svg" alt="tonic" /></div>
+        <div class="block">
+          <h1 class="heading1">Tonic</h1>
+          <div class="subframe">
+            <p class="sub1">CANOPY</p>
+            <div class="circle"></div>
+            <p class="sub2">Back End Dev</p>
+            <div class="circle"></div>
+            <p class="sub2">2015</p>
+          </div>
+          <div class="normal">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </div>
```

Comment:
- [ ] Since this content is a paragraph it more semantically appropriate to use `<p>` tag instead. We reserve <div> to sections that where we can't use semantic section tags such as `section, header, main, footer, `etc.